src/common/AudioStream.cpp

```diff
@@ -39,22 +39,24 @@ Result AudioStream::close() {
     return Result::OK;
 }
 
-DataCallbackResult AudioStream::fireDataCallback(void *audioData, int32_t numFrames) {
-    if (!isDataCallbackEnabled()) {
-        LOGW("AudioStream::%s() called with data callback disabled!", __func__);
-        return DataCallbackResult::Stop; // We should not be getting called any more.
-    }
-
-    // TODO remove
+// Call this from fireDataCallback to monitor CPU scheduler.
+void AudioStream::checkScheduler() {
     int scheduler = sched_getscheduler(0) & ~SCHED_RESET_ON_FORK; // for current thread
     if (scheduler != mPreviousScheduler) {
         LOGD("AudioStream::%s() scheduler = %s", __func__,
              ((scheduler == SCHED_FIFO) ? "SCHED_FIFO" :
-             ((scheduler == SCHED_OTHER) ? "SCHED_OTHER" :
-             ((scheduler == SCHED_RR) ? "SCHED_RR" : "UNKNOWN")))
+              ((scheduler == SCHED_OTHER) ? "SCHED_OTHER" :
```

Comment:
nit: is this the indentation you wanted?

Reply:
Nope. Auto-indenter got creative. Thanks.